app/_components/cbio-util.js

```diff
@@ -666,6 +672,47 @@ cbio.util = (function() {
     return def.promise();
   }
 
+
+  function getDecimalExponents(data){
+
+    // Copy the values, rather than operating on references to existing values
+    if (data === null || data.length < 1) {//if data is null or empty, return data
+      return data;
+    }
+    
+    var values = [];
+    _.each(data, function(item) {
+      if (!isNaN(item)) {
+        values.push(Number(item));
+      }
+    });
+
+    // Then sort
+    values.sort(function(a, b) {
```

Comment:
The default sort() should return values from small numbers to large values. But in here, it works conversely, so I have to write 'a-b'.